lambda/utils.go

```diff
@@ -169,6 +169,23 @@ func GetCredsType(s string) string {
 	}
 }
 
+func ParseJsonSecret(s string) (secret string, err error) {
+	var jsonData map[string]interface{}
+	jsonErr := json.Unmarshal([]byte(s), &jsonData)
+	if jsonErr != nil {
+		return "", fmt.Errorf("json unmarshal error: %v", jsonErr.Error())
+	}
+	username, ok := jsonData["username"].(string)
+	if !ok {
+		return "", fmt.Errorf("json username error")
+	}
+	password, ok := jsonData["password"].(string)
+	if !ok {
+		return "", fmt.Errorf("json password error")
```

Comment:
can you make the errors a bit more descriptive pls?

Reply:
Just updated those error messages. Let me know if you're looking for anything more specific.